svelte.config.js

```diff
@@ -10,7 +10,7 @@ const config = {
 		// hydrate the <div id="svelte"> element in src/app.html
 		target: '#svelte',
 		files: {
-			lib: 'src/components'
+			lib: 'src/lib'
 		}
```

Comment:
`src/lib` is een default value volgens mij. dus dit stukje kan er misschien wel uit?

Reply:
Ur right